azalea-client/src/account.rs

```diff
@@ -128,24 +137,35 @@ impl Account {
     /// # async fn example() -> Result<(), Box<dyn std::error::Error>> {
     /// let client = reqwest::Client::new();
     ///
-    /// let res = azalea_auth::get_ms_link_code(&client).await?;
+    /// let res = azalea_auth::get_ms_link_code(&client, "client_id", "scope").await?;
+    /// // Or, `azalea_auth::get_ms_link_code(&client, Some(client_id)).await?`
```

Comment:
The second line here should be taking two arguments.
```suggestion
    /// let res = azalea_auth::get_ms_link_code(&client, None, None).await?;
    /// // Or, `azalea_auth::get_ms_link_code(&client, Some(client_id), None).await?`
```